src/parser/jobs/war/modules/SurgingTempest.tsx

```diff
@@ -32,6 +83,17 @@ export class SurgingTempest extends Analyser {
 				}),
 			],
 		}))
+		this.suggestions.add(new TieredSuggestion({
+			content: <Trans id="war.stormseye.suggestions.overwrite.content">
+			Avoid using <DataLink action="STORMS_EYE" /> too early, as it does less damage and generates less beast gauge than <DataLink action="STORMS_PATH" />
```

Comment:
It doesn't do less damage, they're both 400 potency. Use proper sentences and be consistent with case tho.
```suggestion
			Avoid using <DataLink action="STORMS_EYE"/> more than {EYE_BUFFER / 1000} seconds before it expires, as it generates less Beast Gauge than <DataLink action="STORMS_PATH"/> which can cost you uses of your gauge consumers.
```